src/Microsoft.Extensions.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -10,8 +10,17 @@
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Static helper class that provides methods to bind types against configuration.
+    /// </summary>
     public static class ConfigurationBinder
     {
+        /// <summary>
+        /// Attempts to bind the instance against configuration using reflection matching
+        /// property names against values recursively.
```

Comment:
Attempts to bind the object instance passed to configuration values by matching property names against configuration keys recursively.